.wokeignore

```diff
@@ -0,0 +1 @@
+docs/.wokeignore
```

Comment:
This change is as suggested by the starter pack, a symlink to the `docs/.wokeignore`.